pages/workflow.md

```diff
@@ -1,18 +1,31 @@
+* First I started by getting the Data from the website metrocosm.com which had an excel sheet that had all of the immigration Data that we needed which was from 1825-2017.
+* I then started to alter the excel sheet by flipping it to make it work for Tableau to make the visualizations.
+* I then made three visualizations which I used area charts for all of them which helped show the results of our data clearer.
+* As soon as I finished making my visualizations the data showed the increases and decreases of migrations to the United States then I showed my group the visualizatiuons and we got to see the results.
 
-To make elegant workflow charts, use [LucidChart](https://lucidchart.com)
+We were lucky enough that we had a source that we did not have to scrape for any data since we already had the data. The source that we had was metrocosm.com which had an excel spread sheet. We had to fix the excel sheet up and then we had to flip the information so it can be readable to Tableau. After that with that information from the data I was able to make three visualizations. 
 
-Take plenty of screenshots to document your process...
-    * **Mac** press 'command' + 'shift' + '4' and then drag a rectangle to take a screenshot of whatever you select... On
-    * **Windows**, click the 'Start' button then type 'Snipping' and select Snipping Tool, then click 'New' and drag a rectangle to take a screenshot of whatever you select.
+The first visualization being the main visualization on immigration from 19th century until 21st century which had some laws that effected immigration throughout the timeline. For example, there was a drop of immigration during the 1917 immigration act, which then the drop kept decreasing from 1917 until events leading to WWII. There was also the 1953 Refugee Relief Act which showed from the graph that little by little throughout decades’ immigrations started to Increase tremendously. The Visualization showed the major shifts of migration to the United States starting from the year 1825, which showed the cause being United States Laws. The second visualization is about immigration Pre-World War II by European nations. The Visualization showed that migration was decreasing by a huge margin leading to World War II which makes sense since Wars against other nations would cause some National security concerns. It first started dropping from the events of World War I but then closer to the second World War there was stricter immigration policies. The last visualization was on wars and events that effected immigration. There was a big growth during 1880 from Asians because of the Angell Treaty of 1880 which helped the growth of Asian migration during the late 1800’s. 
```

Comment:
The portion above should all have been much more expansive, including using screenshots....

But what this portion and the following paragraph (until Research Work Flow) should actually be on the analysis page... since they are essentially talking about the actual visualization. So, they should be right next to the visualization they are talking about.

After all, you really aren't talking about a 'workflow' here, but just talking about what the graph shows. That is analysis, and should belong as the analysis page.